cli/src/commands/content/curatorGroup.ts

```diff
@@ -36,5 +36,11 @@ export default class CuratorGroupCommand extends ContentDirectoryCommandBase {
         )
         .join(', ')
     )
+    displayHeader(`Group Permissions (${[...permissionsByLevel].length})`)
+    this.log(
+      [...permissionsByLevel]
+        .map(([level, permissions]) => chalk.magentaBright(`Privilege Level: ${level}; (Permissions: ${permissions})`))
+        .join(', ')
```

Comment:
This needs some better visual distinction, for example:
```suggestion
        .join('\n\n')
```

Perhaps you could also display `ChannelAgentPermissions` of each group member (curator)?